Source/WebKit/Shared/unix/BreakpadExceptionHandler.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if ENABLE(BREAKPAD)
+#include "config.h"
```

Comment:
You have to `#include "config.h"` first for `ENABLE()` and `ENABLE_BREAKPAD`. This probably only works due to unified build?